.github/workflows/run-tests.yml

```
@@ -0,0 +1,36 @@
+name: Run Unit Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: '*'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10  
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10"]
```

Since the personas package is compatible with any Python version >=3.9, we should test against the current floor and ceiling in that range.
```suggestion
        python-version: ["3.9", "3.13"]
```